app/docker/networks/edit/NetworkDetailsView.tsx

```diff
@@ -0,0 +1,96 @@
+import { useRouter, useCurrentStateAndParams } from '@uirouter/react';
+import DockerNetworkHelper from 'Docker/helpers/networkHelper';
+
+import { useEnvironmentId } from '@/portainer/hooks/useEnvironmentId';
+import { PageHeader } from '@/portainer/components/PageHeader';
+import { confirmDeletionAsync } from '@/portainer/services/modal.service/confirm';
+import * as notifications from '@/portainer/services/notifications';
+// import { AccessControlPanel } from '@/portainer/access-control/AccessControlPanel/AccessControlPanel';
+
+import { removeNetwork, isSystemNetwork } from '../network.service';
+import { useNetwork } from '../queries';
+import { IPConfigs } from '../types';
+
+import { NetworkDetailsTable } from './NetworkDetailsTable';
+
+export function NetworkDetailsView() {
+  const router = useRouter();
+
+  const {
+    params: { id: networkId }, // nodeName
+  } = useCurrentStateAndParams();
+  const environmentId = useEnvironmentId();
+
+  const networkQuery = useNetwork(networkId, environmentId); // 1 object
+
+  const allowRemoveNetwork: boolean =
+    networkQuery.data && !isSystemNetwork(networkQuery.data.Name);
+  const IPV4Configs: IPConfigs =
+    networkQuery.data?.IPAM &&
+    DockerNetworkHelper.getIPV4Configs(networkQuery.data?.IPAM.Config);
+  const IPV6Configs: IPConfigs =
+    networkQuery.data?.IPAM &&
+    DockerNetworkHelper.getIPV6Configs(networkQuery.data?.IPAM.Config);
```

Comment:
if you move all these to the details table, like you did with the general info, will mean that:
1. you can render the details table only if query.data is defined
2. inside the table you don't need to check if it's defined

Reply:
updated